scripts/esbuild-module-check.mjs

```diff
@@ -0,0 +1,9 @@
+import fs from "node:fs";
+
+export const isExternal = (dependency) => {
```

Comment:
This is perfect! Can we rename it to `isCjsPackage`? After that, I think this PR will be good :-)